bin/hanami

```diff
@@ -2,5 +2,5 @@
 require 'bundler'
 require 'hanami/cli/commands'
 
-Bundler.require(:plugins)
+Bundler.require(:plugins) if File.exist?(ENV["BUNDLE_GEMFILE"] || "Gemfile")
```

Comment:
WDYT maybe is better to wrap this line by method `load_plugins`?

Reply:
@davydovanton I considered that too, but I'm not sure yet about that API. `load_plugins` looks like a complete plugins system, which is not there yet.

Reply:
@jodosha agree. We can fix it in future (of course, if we need it)